Modules/blake2module.c

```diff
@@ -379,13 +379,13 @@ class _blake2.blake2s "Blake2Object *" "&PyBlake2_BLAKE2sType"
 static Blake2Object *
 new_Blake2Object(PyTypeObject *type)
 {
-    Blake2Object *self;
-    self = (Blake2Object *)type->tp_alloc(type, 0);
+    Blake2Object *self = PyObject_GC_New(Blake2Object, type);
```

Comment:
Why change this?

Reply:
All the other hashlib objects use the GC protocol.

Reply:
The default `PyType_GenericAlloc` allocator handles it, I don't think this change is needed.

Reply:
PyType_GenericAlloc also automatically track but honestly, I again prefer mimicking the other modules and have a more explicit construction.